<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nombre_lineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>270</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="aregar_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>256</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이름/동</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edad_lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="dong_name_combo">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>90</y>
     <width>114</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <item>
    <property name="text">
     <string>이름/아파트명</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>초원아파트1동</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>초원아파트2동</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>초원아파트3동</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>초원아파트4동</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>초원아파트5동</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>연락처</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
